api/v1alpha1/gitsync_types.go

```diff
@@ -101,9 +101,9 @@ type CommitStatus struct {
 	Hash string `json:"hash"`
 
 	// Synced indicates if the sync went through
-	Synced bool `json:"synced,omitempty"`
+	Synced bool `json:"synced"`
 
-	// SyncTime represents the last time that we attempted to sync this commit (whether or not it succeeded)
+	// SyncTime represents the last time that we attempted to sync this commit (whether it succeeded)
```

Comment:
oh, do you mean that we won't set the time if it doesn't succeed? (my original thought is that we would)

Reply:
oh, I have to admit I don't like the IDE's suggestion. :) If the IDE doesn't like the original one, I recommended another one below.